CHANGELOG.md

```diff
@@ -29,6 +29,7 @@ We refer to [GitHub issues](https://github.com/JabRef/jabref/issues) by using `#
 - We improved the duplicate checker such that different editions of the same publication are not marked as duplicates. [2960](https://github.com/JabRef/jabref/issues/2960)
 
 ### Fixed
+- We fixed the a bug that leaves .sav file after SaveAs [#2947]
```

Comment:
A little typo: fixed a bug  and please inlcude the URL behind like we do with other entries